PowerShellTools/DebugEngine/DebugServiceEventsHandlerProxy.cs

```diff
@@ -153,7 +153,7 @@ public PSCredential ReadSecureStringPrompt(string message, string name)
         /// <param name="fullName">Full name of remote file(mapped into local)</param>
         public void OpenRemoteFile(string fullName)
         {
-            Debugger.OpenRemoteFile(fullName);
+            Debugger.OpenFileInVS(fullName, true);

```

Comment:
I m not sure if we need any extra parameter for that, we should be able to detect which kind of debug session we are currently in, then we give different error message. Let me know if I misunderstand the purpose here.